index.js

```diff
@@ -10,8 +10,8 @@ const consoleWidth = () => {
 
 program
   .version('0.2.0')
-  .arguments('<phrase>')
-  .action((slng) => {
+  .arguments('<phrase> [number]')
+  .action((slng, numberOfResults) => {
```

Comment:
we should really pass the number of results as an OPTION and not an action, this would require some rework of your logic

Reply:
FYI, some help on this https://github.com/tj/commander.js/blob/master/examples/pizza

Reply:
So I was reading https://github.com/tj/commander.js#specify-the-argument-syntax 
it said that angled brackets made it required input and square brackets were optional input.
was I wrong to do it this way then?

Reply:
Duc, the way it's currently set up would be used this way: `slng gucci 3` Which isn't as clear as using it as a .option().

If we do it that way, we can say `slng gucci --results 3` which uses the `--flag value` structure which makes it more clear.

Reply:
@psyduc right, but I think this would be best as an option because that way you can do something like this or what not
`slng gucci -r=5`
`slng "step up" -r 7`
`slgn "boss" --results=1`
`slgn "boss" --results 6`